SDL/Hit.cuh

```diff
@@ -145,6 +145,21 @@ namespace SDL
         int sign_b = (b < 0) ? -1 : 1;
         return sign_a * sign_b * a;
     };
+
+    template<typename TAcc>
+    ALPAKA_FN_ACC ALPAKA_FN_INLINE float calculate_dPhi(const TAcc& acc, float phi1, float phi2)
+    {
+        // Calculate dPhi
+        float dPhi = alpaka::math::abs(acc, phi1 - phi2);
+
+        // Adjust dPhi if it is greater than pi
+        if (dPhi > float(M_PI))
+        {
+            dPhi = dPhi - 2 * float(M_PI);
+        }
+
+        return dPhi;
+    };
```

Comment:
It seems like the two use cases of dPhi are to be used in the "float dR2 = dEta*dEta + dPhi*dPhi;" calculation, which doesn't depend on the sign of dPhi, and used in a cut on "(alpaka::math::abs(acc, dPhi) > 0.1f)" which already has an abs value sign. With that in mind, I think we should just get rid of the abs from this function and normalize it between -pi and pi with no abs. That seems like it would be the most efficient code at least.

Reply:
Makes sense!